### PR TITLE
expand e2ee logging to better debug UISIs

### PR DESCRIPTION
**Pull request by ara4n**
to try to diagnose https://github.com/vector-im/riot-web/issues/11502 and https://github.com/vector-im/riot-web/issues/11504 and others

### CODE REVIEW

File: src/crypto/OlmDevice.js [hunk 1]
```diff
@@ -740,6 +740,8 @@ OlmDevice.prototype.createOutboundGroupSession = function() {
 OlmDevice.prototype.encryptGroupMessage = function(sessionId, payloadString) {
     const self = this;
 
+    console.log(`encrypting msg with megolm session ${sessionId}`);
```
Comment (turt2live) [inline src/crypto/OlmDevice.js]: please use the logger

File: src/crypto/index.js [hunk 2]
```diff
@@ -2296,6 +2296,8 @@ Crypto.prototype._getTrackedE2eRooms = function() {
 
 Crypto.prototype._onToDeviceEvent = function(event) {
     try {
+        console.log(`received to_device ${event.getType()} from: ${event.getSender()} id: ${event.getId()}`);
```
Comment (turt2live) [inline src/crypto/index.js]: logger

**Comment by ara4n** [comment]
test fails look to be unrelated

**Comment by t3chguy** [comment]
The sas verification tests are flaky

**Pull request merged**